it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala

```diff
@@ -31,12 +31,11 @@ trait FakeFileUploadBackend extends BeforeAndAfterAll with ScalaFutures {
   final lazy val fileUploadBackendBaseUrl = s"http://localhost:$backendPort"
 
   override def beforeAll() = {
-    super.beforeAll()
```

Comment:
Why it is not needed?

Reply:
i have removed beforeAll, as when the object runs all related code run anyway there is no need to do beforeAll, but afterAll is needed.